nipype/interfaces/afni/base.py

```diff
@@ -45,6 +45,7 @@ def version():
         """
         try:
             clout = CommandLine(command='afni --version',
+                                resource_monitor=False,
```

Comment:
would something bad happen if `resource_monitor` is left out from here? i.e., does nested resource monitoring work? if not, perhaps it would be good to add this info somewhere.

Reply:
Yes, that's why I had to add all of those `resource_monitor`. Basically, both inner and outer interfaces send traces to the same file (with all the derived effects you might expect from races)

Reply:
Do you think this will bite someone? This problem only happens if there are nested interfaces. I think it is not too much to ask somebody with the skills to do that knowingly that they add the `resource_monitor=True`. I think it is and edge case, isn't it?

Reply:
We could check if the profiler file exists not to run an extra ResourceMonitor...

Reply:
The problem is less going forward (though assuming that everybody's going to be up on the latest details of the optional parameters to `BaseInterface` seems dubious), but the already existing interfaces. Obviously we'll know to fix them in niworkflows/fmriprep/mriqc, but how many others exist?

If we can do such a check to not double-run, that'd be good, IMO.

Reply:
Also, this `Info.version` is a great example of a use that's not obviously nested. It's not nested when I program it, and if I'm writing an interface, it might not occur to me to check whether `Info().version` will itself be running an interface.